geo-data-server/src/main/java/org/gridsuite/geodata/server/GeoDataService.java

```diff
@@ -52,6 +58,18 @@ private Set<String> toCountryIds(Collection<Country> countries) {
         return countries.stream().map(Country::name).collect(Collectors.toSet());
     }
 
+    @PostConstruct
+    public void init() {
+        defaultSubstationsGeoData = new DefaultSubstationGeoDataByCountry();
+        try {
+            ClassLoader classLoader = getClass().getClassLoader();
+            File file = new File(classLoader.getResource("config/substationGeoDataByCountry.json").getFile());
+            defaultSubstationsGeoData = mapper.readValue(file, DefaultSubstationGeoDataByCountry.class);
+        } catch (IOException e) {
+            LOGGER.warn("No default geo data by country found for substation");
```

Comment:
If you already bundle a substationGeoDataByCountry.json with your jar, it cannot happen, just rethrow an wrapper UncheckIoException